src/Contacts/Threads/TSContactThread.m

```diff
@@ -32,7 +32,12 @@ + (instancetype)getOrCreateThreadWithContactId:(NSString *)contactId
         [SignalRecipient recipientWithTextSecureIdentifier:contactId withTransaction:transaction];
 
     if (!recipient) {
-        recipient = [[SignalRecipient alloc] initWithTextSecureIdentifier:contactId relay:relay supportsVoice:YES];
+        recipient =
+            [[SignalRecipient alloc] initWithTextSecureIdentifier:contactId
+                                                            relay:relay
+                                                    supportsVoice:YES
+                                                   // Default to NO; ContactsUpdater will try to update this property.
```

Comment:
If the contactsUpdater does create a SignalRecipient with the correct `supportsWebRTC` value, aren't we clobbering it with a `NO` on line 48? ` [recipient saveWithTransaction:transaction];`

Maybe the save needs to be moved up. Or maybe it can be deleted, since it looks like ContactsUpdater saves the recipient.